diff -Nru libmng-1.0.5.orig/Makefile.am libmng-1.0.5/Makefile.am
--- libmng-1.0.5.orig/Makefile.am	Sat Mar  1 17:09:16 2003
+++ libmng-1.0.5/Makefile.am	Tue Mar  4 09:26:00 2003
@@ -2,8 +2,10 @@
 
 AUTOMAKE_OPTIONS = 1.3 foreign no-dependencies
 
+SUBDIRS = doc
+
 # include the app subdirectories in the distribution
-EXTRA_DIST = makefiles doc contrib
+EXTRA_DIST = makefiles contrib
 
 
 # libmng release @VERSION@
diff -Nru libmng-1.0.5.orig/configure.in libmng-1.0.5/configure.in
--- libmng-1.0.5.orig/configure.in	Sat Mar  1 17:09:15 2003
+++ libmng-1.0.5/configure.in	Tue Mar  4 09:25:11 2003
@@ -187,4 +187,4 @@
 
   test $_restore -eq 1 && CPPFLAGS=$_cppflags LDFLAGS=$_ldflags
 
-AC_OUTPUT(Makefile)
+AC_OUTPUT(Makefile doc/Makefile doc/man/Makefile)
diff -Nru libmng-1.0.5.orig/doc/Makefile.am libmng-1.0.5/doc/Makefile.am
--- libmng-1.0.5.orig/doc/Makefile.am	Thu Jan  1 01:00:00 1970
+++ libmng-1.0.5/doc/Makefile.am	Tue Mar  4 09:25:11 2003
@@ -0,0 +1,4 @@
+## Process this file with automake to produce Makefile.in
+
+SUBDIRS = man
+
diff -Nru libmng-1.0.5.orig/doc/man/Makefile.am libmng-1.0.5/doc/man/Makefile.am
--- libmng-1.0.5.orig/doc/man/Makefile.am	Thu Jan  1 01:00:00 1970
+++ libmng-1.0.5/doc/man/Makefile.am	Tue Mar  4 09:25:11 2003
@@ -0,0 +1,3 @@
+## Process this file with automake to produce Makefile.in
+
+man_MANS = libmng.3 jng.5 mng.5
